scripts/ipfs/uninstall

```diff
@@ -25,3 +25,6 @@ fi
 sudo rm -f /etc/nginx/sites-enabled/ipfs-http-gateway
 sudo rm -f /etc/nginx/sites-available/ipfs-http-gateway
 sudo systemctl restart nginx.service
+
+# Remove a dependant module
+source ../peer-ipfs-bootstrap/uninstall
```

Comment:
We don't normally remove "Depended" modules since i don't think we support removing a single module